tree/tree/inc/TLeafElement.h

```diff
@@ -34,12 +34,17 @@ class TLeafElement : public TLeaf {
    Int_t               fID;           ///<  element serial number in fInfo
    Int_t               fType;         ///<  leaf type
 
+private:
+   virtual Int_t       GetOffsetHeaderSize() const {return 1;}
+
+   Bool_t              fUnknownDataSize; ///<! Set to true if we don't know the data size for offset arrays.
```

Comment:
I am leery of introducing another data member.  Could fLenType == 0 serve the same purpose?

Reply:
I think that could work, yes.